File: src/main/java/seremis/geninfusion/core/GICreativeTab.java
package seremis.geninfusion.core;

import cpw.mods.fml.relauncher.Side;
import cpw.mods.fml.relauncher.SideOnly;
import net.minecraft.creativetab.CreativeTabs;
import net.minecraft.item.Item;
import seremis.geninfusion.item.ModItems;

public class GICreativeTab extends CreativeTabs {
    public GICreativeTab(String name) {
        super(name);
    }
    
	@Override
    @SideOnly(Side.CLIENT)
	public Item getTabIconItem() {
		return ModItems.titaniumIngot;
	}

}

File: src/main/java/seremis/geninfusion/core/GICreativeTab.scala
package seremis.geninfusion.core

import cpw.mods.fml.relauncher.{Side, SideOnly}
import net.minecraft.creativetab.CreativeTabs
import net.minecraft.item.Item
import seremis.geninfusion.block.ModBlocks

class GICreativeTab(name: String) extends CreativeTabs(name) {

    @SideOnly(Side.CLIENT)
    override def getTabIconItem: Item = Item.getItemFromBlock(ModBlocks.crystal)
}